src/test_l1_plat_power.c

```diff
@@ -522,7 +550,7 @@ void test_l1_plat_power_positive_PLAT_API_SetWakeupSrc (void)
 
     // Variation/Step 20: Call PLAT_TERM() - close interface
     ret = PLAT_TERM();
-    UT_ASSERT_EQUAL_FATAL(ret, PWRMGR_SUCCESS); // Ensure the returned value is PWRMGR_SUCCESS
+    UT_ASSERT_EQUAL(ret, PWRMGR_SUCCESS); // Ensure the returned value is PWRMGR_SUCCESS
```

Comment:
line 542 why is it commented?

Reply:
@thanushreevani ,  NOT supporting any specific wake up source is not an issue in this scenario.  The aim is to check that when you try to set a wakeup src that is still supported on this platform and fails for a different reason, then the test will terminate at line 546.  If you are satisfied with this answer please do close this issue so that we can keep the things ready to test.